css/css-grid/layout-algorithm/grid-as-flex-item-should-not-shrink-to-fit-001.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Evaluate the behavior of a grid container as Flexbox item</title>
+<link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#layout-algorithm">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#flex-lines">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#propdef-flex-wrap">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#propdef-align-self">
+<link rel="help" href="https://drafts.csswg.org/css-align-3/#align-flex">
+<link rel="help" href="https://drafts.csswg.org/css-align-3/#align-grid">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht">
+<meta name="assert" content="The 'auto' inline-size of the grid item should fill the available space of its grid container, which shouldn't shrink wrap honoring the 'align-self: normal' (behaves as 'stretch') as flexbox item, despite the default behavior of single-line flexbox.">
+<link rel="stylesheet" href="/fonts/ahem.css">
+<style>
+    body { overflow: hidden; }
+    .flexbox {
+        display: flex;
+        flex-flow: column nowrap;
+        width: 100px;
+        height: 100px;
+        font: 25px/1 Ahem;
+    }
+    .grid {
+        display: grid;
+        align-items: baseline;
+        background: red;
+        color: green;
```

Comment:
Shouldn't color and font be in the gridItem class? I think it's a bit less confusing. Same for the other tests.

Reply:
I use to set the font in the top container, so that it applies to all the grid items. I can move the color to the flexbox container as well if you consider it more convenient.